### PR TITLE
Added reference case to kunder-och-expertis.

### LINKED ISSUE #91: Info om tidigare uppdrag till kundsidan
Detta handlar om kundsidan, dvs. https://etimo.se/kunder-och-expertis

Sektion "Tjänster" ska flyttas (1 i bilden) och de två sektionerna "Matsvinn" och "Migrering till molnet" tillkommer i detta ärende (2 i bilden). När ärendet är klart är alltså vänstermenyn på kundsidan såhär:
image

**Sektion "Matsvinn"**
image
Design kan kopieras från sektion "Vår passion" på startsidan. Text i högra delen (notera "text-align: justify;"):

> Kunden är en organisation som arbetar för att minska matsvinn. Uppdraget bestod i att designa och skriva om deras existerande lösning för hantering av betalningar. Detta för att möta nya krav från PSD2 och för att leverera en mer säker betalningsupplevelse till sina kunder. I samband med omskrivningen gjordes flera designförändringar för att lättare kunna spåra betalningar.
> Teknik som användes var Node och Typescript.
> 
> Citat från kunden:
> Etimo joined us on a project basis, taking on a complex system rewrite. He proved a high level of seniority, asking thorough questions and delivered a lot to the technical design choices of the system. Etimo delivered on time, improving the overall technical maturity of business. He also, through well-written code, mentored more junior developers to a higher level of expertise.

**Sektion "Migrering till molnet"**
image
Design kan kopieras från sektion "Vår passion" på startsidan. Text i högra delen (notera "text-align: justify;"):

> Detta uppdrag handlade om DevOps-aktiviteter vilket vår konsult tyckte var en rolig variation. Kunden är i en moderniseringsfas där man går från on-prem till molnet, både för utveckling och drift. De anlitade Etimo för en stor del av denna migrering.
> Teknik som användes under uppdraget var Azure DevOps, Git, C#, Visual Studio, vsix-paket för custom release tasks och TFS.
> 
> Resultat:
> Kunden övergick till mer moderna molnbaserade verktyg och återanvändbara komponenter vilket har höjt effektiviteten och har frigjort mycket tid från driftavdelningen. Nu kan byggen och releaser sättas upp mycket snabbt och med högre kvalitet. Modifieringar som tidigare var omfattande och tidsödande görs nu i en enda av de återanvändbara komponenterna.

**Annan info**
Tänk på att varannan sektion på sidan ska ha grå bakgrund (även om det inte framgår av bilderna i detta ärende).

### PR DESCRIPTION
**Pull request by ForsbergPhilip**
* Added "Referenscase" to kunder-och-expertis.
* Added information about "Migrering to molnet" in new ReferenceCase component.
* Changed position of "Kompetenser" and "Tjänster" in scroller.
Closes #91

### CODE REVIEW

File: src/components/Clients/ReferenceCase.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import tw from 'twin.macro';
+import FadeIn from '../../animations/FadeIn';
+import Caption from '../../elements/Caption';
+import H2 from '../../elements/H2';
+import P from '../../elements/P';
+import Span from '../../elements/Span';
+import { useViewportSize } from '../../hooks';
+import Section from '../Section';
+
+const EmphasizedH2 = styled(H2)`
```
Comment (H4zze) [inline src/components/Clients/ReferenceCase.tsx:12]: This const is reused in multiple places, maybe we could move it to one place and import it.

Comment (ForsbergPhilip) [reply src/components/Clients/ReferenceCase.tsx]: Moved all EmphasizedH2 that uses the same css to a new element.

**Comment by joakim-olesen** [comment]
Section backgrounds should alternate between gray and white. Now, two sections have the same background (both section "Värde för kund" and "Kompetenser" have white backgrounds).

**Comment by ForsbergPhilip** [comment]
> Section backgrounds should alternate between gray and white. Now, two sections have the same background (both section "Värde för kund" and "Kompetenser" have white backgrounds).

Great catch! I have fixed this now.

**Comment by joakim-olesen** [comment]
Now "Referenscase" and "Våra kunder" both have a grey background. But I think that's fine because the latter section should be moved in ticket https://github.com/Etimo/homepage/issues/108 anyways so no reason to fix more section backgrounds in this ticket :)

The page looks good!

**Comment by ForsbergPhilip** [comment]
> Now "Referenscase" and "Våra kunder" both have a grey background. But I think that's fine because the latter section should be moved in ticket #108 anyways so no reason to fix more section backgrounds in this ticket :)
> 
> The page looks good!

Didn't see that because clicking Våra kunder sends me to "Kontakta oss" section. Will change so clicking on "Våra kunder" send us to the correct section. Fixing the background as well.

**Review by joakim-olesen — APPROVED** [review]

**Pull request merged**